MinStack.java

```diff
@@ -0,0 +1,93 @@
+// Time Complexity : O(n)
+// Space Complexity :O(n)
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : is time complexity right for this problem i am a bit confused.
```

Comment:
Yes it is O(n) because you searched thru all the elements in the stack
The question asks to complete it in constant time